test/sandbox/config/environments/test.rb

```diff
@@ -1,5 +1,13 @@
 # frozen_string_literal: true
 
+require "warning"
+
+# Use https://github.com/jeremyevans/ruby-warning
+# to restrict warnings outside our control
+["mail", "activesupport", "yard"].each do |gem_name|
```

Comment:
I also saw this in the README if we wanted to ignore all gems, but no strong feelings:

```
# Ignore all warnings in Gem dependencies
Gem.path.each do |path|
  Warning.ignore(//, path)
end
```

Reply:
I was thinking would be good to ignore as little as possible to start as I'm worried about missing legitimate warnings.